benchmarks/array/main.cu

```diff
@@ -116,21 +116,21 @@ int main(int argc, char** argv) {
 
         uint64_t page_size = settings.pageSize;
         uint64_t n_pages = settings.numPages;
-        uint64_t total_cache_size = (page_size * n_pages);
+        //uint64_t total_cache_size = (page_size * n_pages);
```

Comment:
I propose moving towards fewer lines of commented out code, this would greatly improve the readability of the code base.
In my opinion, it would be better to simply remove the lines you're commenting out.

Reply:
Totally agreed! Since we already have full modification log in the git repo. I personally would prefer to only leave effective code too. I've removed some of it in other refactoring commit. For this commit, it's just try to temporary fix the compile warnings. I choose to do it against my prefer because of two reasons:
1. Original BaM code base already have lots of comment-out / deprecated codes. It seems original author prefer to keep the "possible for future use" code there. I'm just following the original style.
2. For "benchmarks" codes, it looks like some testing codes that leaves others to modify/tune the code to fit their benchmark scenario.

If the repo maintainer would like to just remove it, I surely can do it. :)